training-maquetado/index.html

```diff
@@ -133,5 +138,12 @@
       </div>
     </article>
   </section>
+  <footer class="footer d-flex align-center justify-center fd-column">
+    <h3>we are game changers</h3>
+    <img class="footer-logo" src="./assets/wolox_logo_white.svg" alt="Wolox Logo">
+    <small>
```

Comment:
I would add this a class as if for some reason in the future you would need to add another small element you would need to change the style of this one.